spec/services/linked_data/graph_service_spec.rb

```diff
@@ -103,26 +103,8 @@
           .to_return(status: 200, body: webmock_fixture('lod_lang_search_enfrde.rdf.xml'), headers: { 'Content-Type' => 'application/rdf+xml' })
       end
 
-      context 'and one language passed in as string' do
-        let(:language) { "en" }
```

Comment:
This was testing normalization of language.  That processing moved to LanguageService and is tested there.